frontends/web/src/components/rates/rates.tsx

```diff
@@ -42,7 +42,7 @@ export const currencies: Fiat[] = ['AUD', 'BRL', 'CAD', 'CHF', 'CNY', 'EUR', 'GB
 
 export const store = new Store<SharedProps>({
     rates: undefined,
-    active: 'CHF',
+    active: 'USD',
```

Comment:
I don't remember now: why is the default fiat set in both backend and frontend. shouldn't backend setting be enough?

Reply:
Yes the setting is coming from the backend, this is only the default initialization of the custom frontend Store.ts.